docs/tools/libzbc.md

```diff
@@ -258,9 +260,9 @@ Zone 55878: type 0x2 (Sequential-write-required), cond 0x1 (Empty), reset recomm
 Zone 55879: type 0x2 (Sequential-write-required), cond 0x1 (Empty), reset recommended 0, non_seq 0, sector 29296689152, 524288 sectors, wp 29296689152
 ```
 
-The output of *zbc_report_zones* can also be limited using the option `-ro`
-(reporting options) for filtering the zones reported, or the option `-n` to
-limit the output to the number of zones that would be reported.
+The output of *zbc_report_zones* can be limited by using the option `-ro`
+(reporting options) to filter which zones are reported, or by using the option
+`-n` to limit the output to the number of zones reported.
```

Comment:
Bug: this should be: "...to limit the output to the number of zones specified by this option."

Reply:
I've gone slightly more pedantic:

> ...to limit the output to the number of zones specified as an argument of
> this option.